Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -0,0 +1,319 @@
+STEP_INIT = "init"
+
+
+class RecoClient:
+    def __init__(self, client: BaseClient):
+        self.client = client
+
+    def get_incidents(
+            self,
+            risk_level: Optional[int] = None,
+            source: Optional[str] = None,
+            before: Optional[datetime] = None,
+            after: Optional[datetime] = None,
+            limit: int = 1000,
+    ) -> List[Dict[str, Any]]:
+        """
+        Fetch incidents from Reco API
+        :param risk_level: The risk level of the incidents to fetch
+        :param source: The source of the incidents to fetch
+        :param before: The maximum date of the incidents to fetch
+        :param after: The minimum date of the incidents to fetch
+        :param limit: int
+        :return: dict
+        """
+        demisto.info("Get incidents, enter")
+        alerts: List[Dict[str, Any]] = []
+        params: Dict[str, Any] = {
+            "getTableRequest": {
+                "tableName": RECO_ACTIVE_INCIDENTS_VIEW,
+                "pageSize": limit,
+                "fieldFilters": {
+                    "relationship": FILTER_RELATIONSHIP_AND,
+                    "filters": {"filters": []},
+                },
+            }
+        }
+        if risk_level:
+            params["getTableRequest"]["fieldFilters"]["filters"]["filters"].append(
+                {"field": "risk_level", "stringEquals": {"value": risk_level}}
+            )
+        if source:
+            params["getTableRequest"]["fieldFilters"]["filters"]["filters"].append(
+                {"field": "data_source", "stringEquals": {"value": source}}
+            )
+        if before:
+            params["getTableRequest"]["fieldFilters"]["filters"]["filters"].append(
+                {
+                    "field": CREATED_AT_FIELD,
+                    "before": {"value": before.strftime("%Y-%m-%dT%H:%M:%SZ")},
+                }
+            )
+        if after:
+            params["getTableRequest"]["fieldFilters"]["filters"]["filters"].append(
+                {
+                    "field": CREATED_AT_FIELD,
+                    "after": {"value": after.strftime("%Y-%m-%dT%H:%M:%SZ")},
+                }
+            )
+
+        demisto.debug(f"params: {params}")
+        try:
+            response = self.client._http_request(
+                method="PUT",
+                url_suffix="/incident",
+                data=json.dumps(params),
+                timeout=RECO_API_TIMEOUT,
+            )
+
+            if response.get("getTableResponse") is None:
+                demisto.info(f"got bad response, {response}")
+            else:
+                demisto.info(
+                    f"Count of incidents: {response.get('getTableResponse').get('totalNumberOfRecords')}"
+                )
+                alerts = (
+                    response.get("getTableResponse", {}).get("data", {}).get("rows", [])
+                )
+                demisto.info(f"Got {len(alerts)} alerts")
```

Comment:
Alerts? You're mixing vocabularies (this happens a lot in the code, only commenting here)

Use a single word per concept